src/CabanaPD_Particles.hpp

```diff
@@ -726,6 +729,14 @@ class Particles<MemorySpace, PMB, TemperatureDependent, Dimension>
     {
         return Cabana::slice<0>( _aosoa_temp, "temperature" );
     }
+    auto sliceTemperatureAtomic()
```

Comment:
What does sliceTemperatureAtomic compute exactly? Is temp_a the temperature of a particle or the temperature of all particles in a processor or something else?

Reply:
It's the same particle data as `sliceTemperature`, but with protection against multiple threads modifying at the same time

Reply:
OK, thanks.